src/brogue/IO.c

```diff
@@ -4550,13 +4550,8 @@ void highlightScreenCell(short x, short y, color *highlightColor, short strength
     storeColorComponents(displayBuffer[x][y].backColorComponents, &tempColor);
 }
 
-short estimatedArmorValue() {
```

Comment:
Can we document what this represents, or is for? I guess this is for the sidebar?

Reply:
Added a comment describing what it computes.